browser/ui/webui/basic_ui.h

```diff
@@ -42,18 +42,7 @@ class BasicUI : public content::WebUIController {
           bool disable_trusted_types_csp = false);
   ~BasicUI() override;
 
-  // Called when subclass can set its webui properties.
-  virtual void UpdateWebUIProperties() {}
-
- protected:
-  bool IsSafeToSetWebUIProperties() const;
-  content::RenderFrameHost* GetRenderFrameHost();
```

Comment:
forward declaration for `content::RenderFrameHost` is not needed

Reply:
done.